src/Tribe/PUE/Notices.php

```diff
@@ -212,6 +212,15 @@ public function render_invalid_key() {
 			return;
 		}
 
+		// Only show our invalid key messaging on the plugin admin screen and on our own admin screens
+		// @todo review and revise in MR 17.13
+		if (
+			'plugins.php' !== $pagenow
```

Comment:
Can't we use the `is_screen` for that? By passing the page id you can use that helper class for this condition